app/_how-tos/operator-controlplane-feature-gates-controllers.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Configure ControlPlane Feature Gates and Controllers
+description: "Configure feature gates and controllers for Kong ControlPlane in {{ site.ko_product_name }}"
+content_type: how_to
+
+permalink: /operator/control-planes/how-to/configure-feature-gates-controllers/
+breadcrumbs:
+  - /operator/
+  - index: operator
+    group: Control Plane
+  - index: operator
+    group: Control Plane
+    section: "How-To"
+
+products:
+  - operator
+
+works_on:
+  - on-prem
+
+entities: []
+
+tags:
+  - controlplane
```

Comment:
You can add the words in `.github/style/base/Dictionary.txt` to add  them to dictionary used by Vale.

Reply:
Thank you!